src/Public.Api/Address/BackOffice/AddressBackOfficerController-CorrectBoxNumber.cs

```diff
@@ -0,0 +1,110 @@
+
+namespace Public.Api.Address.BackOffice
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using AddressRegistry.Api.BackOffice.Abstractions.Requests;
+    using Be.Vlaanderen.Basisregisters.Api.Exceptions;
+    using Common.Infrastructure;
+    using Infrastructure;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.AspNetCore.Mvc.Infrastructure;
+    using RestSharp;
+    using AddressRegistry.Api.Legacy.Address.Responses;
+    using Infrastructure.Swagger;
+    using Newtonsoft.Json;
+    using Swashbuckle.AspNetCore.Annotations;
+    using Swashbuckle.AspNetCore.Filters;
+    using ProblemDetails = Be.Vlaanderen.Basisregisters.BasicApiProblem.ProblemDetails;
+
+    public partial class AddressBackOfficeController
+    {
+        public const string CorrectBoxNumberRoute = "adressen/{objectId}/acties/corrigeren/busnummer";
+
+        /// <summary>
+        /// Corrigeer het busnummer van een adres.
+        /// </summary>
+        /// <param name="objectId">Identificator van het adres.</param>
+        /// <param name="addressCorrectBoxNumberRequest"></param>
+        /// <param name="actionContextAccessor"></param>
+        /// <param name="problemDetailsHelper"></param>
+        /// <param name="changeBoxNumberAddressToggle"></param>
+        /// <param name="ifMatch">If-Match header met ETag van de laatst gekende versie van het adres (optioneel).</param>
+        /// <param name="cancellationToken"></param>
+        /// <response code="202">Als het busnummer van een adres succesvol gecorrigeerd is.</response>
```

Comment:
Can you change docs to what is in PR https://github.com/Informatievlaanderen/public-api/pull/514 ?